src/transformers/models/paligemma/processing_paligemma.py

```diff
@@ -97,7 +108,10 @@ def __init__(self, image_processor=None, tokenizer=None):
         image_token = AddedToken(IMAGE_TOKEN, normalized=False, special=True)
         tokens_to_add = {"additional_special_tokens": [image_token]}
         tokenizer.add_special_tokens(tokens_to_add)
+        tokenizer.add_tokens(EXTRA_TOKENS)
```

Comment:
add twice is slow but it's okay. 
```suggestion
        tokenizer.add_tokens([image_token]+EXTRA_TOKENS)
```

Reply:
but leave it as is is fine